src/Form/Type/ImmutableTabsType.php

```diff
@@ -24,14 +24,18 @@
  *
  * @author Benoit de Jacobet <benoit.de-jacobet@ekino.com>
  */
-class ImmutableTabsType extends ImmutableArrayType
+class ImmutableTabsType
```

Comment:
extending `AbstractType` is kind of mandatory.

Reply:
AbstractType does not provide implementation for configureOption so if I go back to inheritance I implement it here. I have to see which values to allow though